app/plugin_api.go

```diff
@@ -381,6 +381,10 @@ func (api *PluginAPI) GetProfileImage(userId string) ([]byte, *model.AppError) {
 	return data, err
 }
 
+func (api *PluginAPI) GetEmojiList(page, perPage int) ([]*model.Emoji, *model.AppError) {
+	return api.app.GetEmojiList(page, perPage, "")
```

Comment:
Although not exposed in the client API, I'm wondering if we should go ahead and expose the `sort` parameter here explicitly, mimicking `GetUsersInChannel` above (`sortBy string, page, perPage int`). The only supported sort is `model.EMOJI_SORT_BY_NAME`, so it might seem superfluous for now, but it avoids a future API break to add the parameter later.

Reply:
Ok! I have added the requested changes in the new commit.